tests/show.rs

```diff
@@ -89,6 +90,7 @@ latest-{target} (default)
 "#
         );
         assert!(stdout.contains(expected_stdout));
+        assert!(!stdout.contains("fuels versions"));
```

Comment:
The heading "fuels versions" is a little confusing to me because I think of `fuels` as the SDK, not forc. Perhaps a better heading would be "Fuel component versions" ?

Reply:
Or even just "Component versions" or "Package versions" since the use is running fuelup, so the versions much be pertaining to fuel.

Reply:
> The heading "fuels versions" is a little confusing to me because I think of fuels as the SDK

Yeah actually, that is the point! I think there was a past version of `forc` that deviated from `fuels` which resulted in incompatibility between certain SDK methods and forc itself. So the feature introduced in this PR is meant to serve as a sort of 'guide' as to what fuels version the component was tested/built against.

I agree though it might need better naming or more explanation behind what 'fuels versions' mean :thinking: